Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -39,6 +39,7 @@ however, it has to be formatted properly to pass verification tests.
 - Fixed "Listen" functionality for selecting an input sometimes expecting the wrong input type.
 - Fixed console errors that can be produced when opening input package settings from the Inspector.
 - Fixed InputManager.asset file growing in size on each Reset call.
+- Fixed Opening InputDebugger throws 'Action map must have state at this point' error
```

Comment:
Since this is a public issue provide ISXB-737 reference with hyperlink to public issue tracker. Also missing full-stop at end of line.

Reply:
It was reported internally (not public) and doesn't have any issuetracker page.
Should we make it public?

Reply:
Ah, sorry then we are all good. Assumed it was external since it was an ISXB, but then leave it as-is.